recipes/_install-windows.rb

```diff
@@ -102,6 +104,8 @@ def unmute_host(context)
   # custom credentials and use them if that's the case.
   install_options.concat(' DDAGENTUSER_NAME=%DDAGENTUSER_NAME%') if ddagentuser_name
   install_options.concat(' DDAGENTUSER_PASSWORD=%DDAGENTUSER_PASSWORD%') if ddagentuser_password
+
+  install_options.concat(" NPM=#{dd_agent_install_npm}") if dd_agent_install_npm
```

Comment:
```suggestion
  install_options.concat(" NPM=#{dd_agent_install_npm}") unless dd_agent_install_npm.nil?
```
Nit: Checking for null explicitly. This way if someone sets `windows_npm_install` to `false` we will still pass it as `false` instead of not passing it. Right now this won't do much since `false` is the default, but that might change in the future.